src/core/raster/qgscolorrampshader.h

```diff
@@ -123,11 +123,14 @@ class CORE_EXPORT QgsColorRampShader : public QgsRasterShaderFunction
     bool isEmpty() const;
 
     /**
-     * Gets the source color ramp
+     * Returns the source color ramp.
+     *
+     * Ownership remains with the shader.
```

Comment:
Do we state this?
I think this is always the default assumption unless stated otherwise.

Reply:
Good point. I'll address this in a follow up (easier to backport that way)